grails-app/views/user/show.gsp

```diff
@@ -187,7 +187,14 @@
 
     <div id="map"
          markers-url="${createLink(controller: "user", action: 'ajaxGetPoints', id: userInstance.id)}"
-         infowindow-url="${createLink(controller: 'task', action: 'details')}">
+         infowindow-url="${createLink(controller: 'task', action: 'details')}"
+    <g:if test="${isValidator}">
```

Comment:
Embedding a grails tag inside the <div> opening tag does my head in, you could move it out and g:set a variable instead?

Reply:
OMG how did I write this. Thanks, will fix up.